app/src/main/java/com/example/lastfm/ArtistInfoFragment.java

```diff
@@ -0,0 +1,115 @@
+package com.example.lastfm;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.lastfm.artist_info.ArtistInfo;
+import com.example.lastfm.artist_info.dto.ArtistInfoDto;
+import com.squareup.picasso.Picasso;
+
+import io.reactivex.SingleObserver;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+
+
+
+public class ArtistInfoFragment extends Fragment {
+    private TextView tvNameView;
+    private TextView tvPlayCount;
+    private ImageView artistImage;
+    private TextView tvArtistBio;
+    //Вызывается массив картинок размер которых варьируется от 0-4
+    //0 - самое маленькое разрешение
+    //4 - самое большое разрешение
```

Comment:
move comments to another class

Reply:
moved comments with const to another class